src/wazuh_db/wdb.h

```diff
@@ -1985,16 +1915,6 @@ int wdb_parse_task_delete_old(wdb_t* wdb, const cJSON *parameters, char* output)
  */
  int wdb_parse_agents_remove_vuln_cves(wdb_t* wdb, char* input, char* output);
```

Comment:
The doxygen for this method should be updated

Reply:
Good catch! thanks